go/vt/tabletserver/schema_info.go

```diff
@@ -409,6 +410,10 @@ func (si *SchemaInfo) DropTable(tableName string) {
 
 // GetPlan returns the ExecPlan that for the query. Plans are cached in a cache.LRUCache.
 func (si *SchemaInfo) GetPlan(ctx context.Context, logStats *LogStats, sql string) *ExecPlan {
+	span := trace.NewSpanFromContext(ctx)
+	span.StartLocal("SchemaInfo.GetPlan")
+	defer span.Finish()
+
```

Comment:
Its the locking I was worried about, under load, and non-bindvar queries... Since spans are very light weight, cant hurt :)